lib/gssapi/lib_gssapi_loader.rb

```diff
@@ -23,7 +26,16 @@ def self.load_mit
         gssapi_lib = 'libgssapi_krb5.so.2'
         ffi_lib gssapi_lib, FFI::Library::LIBC
       when /darwin/
-        gssapi_lib = '/usr/lib/libgssapi_krb5.dylib'
+        prefix = begin
+          stdout, _stderr, status = Open3.capture3("krb5-config --prefix")
+          stdout.chomp!
+
+          stdout if stdout != '/' && status == 0
+        rescue Errno::ENOENT
+          # No krb5-config command found, use the default prefix.
```

Comment:
You could just put `"/usr"` in here right? instead of returning nil and then using `||` to default to `"/usr"`?

Reply:
If I understand correctly, then it would be left as `nil` if the exit `status` is non-zero, since the `begin` block returns `nil` in that case.

Reply:
To be clearer, if the condition evaluates to false (i.e. if `stdout == '/'` or `status != 0`), then the "try" section of the `begin` block would evaluate to `nil`, and the `rescue` block only evaluates if an exception was thrown, which will most likely only happen if the `krb5-config` command isn't found, so that would cause the entire `begin` block to evaluate to `nil`. This is why I have to OR the whole block with the default value.

Reply:
Got it. Sounds good.